manager/sql/databases.go

```diff
@@ -0,0 +1,377 @@
+package sql
+
+import migrate "github.com/rubenv/sql-migrate"
+
+type Database struct {
```

Comment:
Please rename this to `sqlStatements` - exporting it is not required because it's an internal struct

Reply:
In order to support external additions of databases which are not officially supported, this would need to remain exported. In that case, would you prefer `SQLStatements`?

Reply:
Yes, I think `Statements` should actually be enough!